.pre-commit-config.yaml

```
@@ -0,0 +1,28 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
+    hooks:
+      - id: check-yaml
+      - id: trailing-whitespace
+        exclude: "(.*\\.(po|pot|svg|ui)$|test/data/.*)"
+      - id: end-of-file-fixer
+        exclude: "(.*\\.(po|pot|svg|ui)$|test/data/.*)"
+      - id: mixed-line-ending
+        exclude: "(.*\\.(po|pot|svg|ui)$|test/data/.*)"
+  - repo: https://github.com/rstcheck/rstcheck
+    rev: v6.2.5
+    hooks:
+    - id: rstcheck
+      args: [--report-level=warning]
```

```suggestion
      args: [--report-level=warning, "--ignore-directives=youtube,vimeo"]
```

The `sphinxcontrib-youtube` plugin adds two new directives `youtube` and `vimeo`.

The `youtube` directive is used in `tutorials/v_attach_disc_id.rst`, `tutorials/v_introduction.rst` and `tutorials/v_submit_acoustid.rst`, which was causing this check to fail.